CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# 1.11.0 (November 14th, 2025)
+
+- Bump MSRV to 1.57 (#788)
+
+### Fixed
+
+- Guarantee address in `slice()` for empty slices. (#780)
```

Comment:
I don't know if this is a fix. It's just a new guarantee, not a bug fix. Perhaps move it under "changes".